docs/release-notes/2.0.0/release-2.0.0.md

```diff
@@ -0,0 +1,198 @@
+# .NET for Apache Spark 2.0.0 Release Notes
+
+### Deprecation for Spark 2.3
+
+We have officially dropped support for Spark 2.3 in the 2.0.0 release. The last Spark 2.3 release (2.3.4) was back in September 2019, and no new release is planned for Spark 2.3. Since there have been no new features introduced for Spark 2.3 in the last few releases of .NET for Apache Spark, if you are relying on Spark 2.3, you should be able to continue using .NET for Apache Spark 1.x.
+
+### New Features/Improvements
+
+* Support for Spark 3.1.0 APIs ([#886](https://github.com/dotnet/spark/pull/886), [#887](https://github.com/dotnet/spark/pull/887), [#888](https://github.com/dotnet/spark/pull/888), [#889](https://github.com/dotnet/spark/pull/889), [#890](https://github.com/dotnet/spark/pull/890), [#893](https://github.com/dotnet/spark/pull/893))
+* Add `Exists(string path)` to `Microsoft.Spark.Hadoop.Fs.FileSystem` ([909](https://github.com/dotnet/spark/pull/909))
+* Add `Version()` to `Microsoft.Spark.SparkContext` and `Microsoft.Spark.Sql.SparkSession` ([919](https://github.com/dotnet/spark/pull/919))
+* Add Avro `ToAvro`, `FromAvro` APIs ([#805](https://github.com/dotnet/spark/pull/805))
+* Extensions.DotNet.Interactive add ENV var to control disposal of tmp dir ([#952](https://github.com/dotnet/spark/pull/952))
+* Add public APIs to access `JvmBridge`, `JvmObjectReference`, `IJvmObjectReferenceProvider` ([#951](https://github.com/dotnet/spark/pull/951))
+  > This is exposed to help users interact with the JVM. It is provided with limited support and should be used with caution.
+
+### Bug Fixes
+
+* Add 'Z' to the string format in Timestamp.ToString() to indicate UTC time ([#897](https://github.com/dotnet/spark/pull/897))
+
+### Infrastructure / Documentation / Etc.
+
+* None
+
+### Breaking Changes
+
+* None
```

Comment:
#897 was a breaking change and that's the reason we are bumping to 2.0?

Reply:
Should we move it here instead ?

Reply:
Moved here.